guides/user/pipeline/pipeline-templates/override.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: single
+title:  "Inherit from the Template or Override It"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In a pipeline that instantiates a template, besides providing values for
+template variables you can...
+
+* [Inherit](#inherit-from-the-template) features from the template
+* [Override](#override-the-template) (add to) inherited features.
+
+## Inherit from the template
+
+By default, the pipeline instance does not inherit items defined in the
+template. You have to explicitly identify what in the template you want to
+inherit. 
+
+For example, the template might have a trigger defined in the
+`"triggers"` element, but that trigger is not used in your pipeline unless you
+include `"triggers"` inside the `"inherit"` element.
+
+1. In the pipeline template you are instantiating, examine what's in the
+`"pipeline"` element to see what you want to use in your pipeline.
+
+   ```bash
+   spin pipeline-template get --id <templateName>
+   ```
+
+1. In the `"inherit": [] ` section of the pipeline JSON, include the key for
+each element of the template that you want to inherit.
+
+   For example, to inherit triggers defined in `pipeline.triggers`, include
+   `"triggers"` inside the `"inherit"` element of the new pipeline:
+
+   ```json
+   "inherit": ["triggers"]
+   ```
+
+   The same goes for anything else found inside `"pipeline"`.
+
+   ```json
+   "inherit": ["triggers", "notifications", "stages"] # and so on...
```

Comment:
"stages" are inherited by default, so they don't belong in this list. Maybe it does make sense to change this behavior to include by default and using an `excludes` in the pipeline config...